src/styles/index.scss

```diff
@@ -1,3 +1,83 @@
 body {
   margin: 0;
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.stopwatch {
+  $minutes-diraction: 3600;
+  $seconds-diraction: 60s;
```

Comment:
The variable `$seconds-diraction` has a typo and a unit redundancy. It should be `$seconds-duration` and you don't need to specify 's' for seconds as SCSS will treat numbers as duration in seconds by default.